src/conf_mode/load-balancing-inbound-ipvsadm.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2020 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+
+from sys import exit
+
+from vyos.config import Config
+from vyos.configdict import dict_merge
+from vyos.template import render
+from vyos.util import cmd
+from vyos.util import dict_search
+from vyos.template import bracketize_ipv6
+from vyos.template import is_ipv6
+from vyos.template import is_ipv4
+from vyos.xml import defaults
+from vyos import ConfigError
+from vyos import airbag
+airbag.enable()
+
+lvs_config = '/tmp/ipvsadm.rules'
+
+def get_config(config=None):
+    if config:
+        conf = config
+    else:
+        conf = Config()
+    
+    base = ['load-balancing', 'inbound', 'ipvsadm']
+    conf = conf.get_config_dict(base, key_mangling=('-', '_'), get_first_key=True)
+    
+    if 'rule' in conf:
+        default_values = defaults(base + ['rule'])
+        for rule in conf['rule']:
+            conf['rule'][rule] = dict_merge(default_values, conf['rule'][rule])
+    
+    return conf
+
+def verify(config):
+    if 'rule' not in config:
+        return None
+    
+    ip_type = None
+    virtual_server_list = []
+    vip_list = []
+    for rule, rule_config in config['rule'].items():
+        if 'disable' in rule_config:
+            continue
+            
+        err_msg = f'Error in the configuration of rule {rule}. '
```

Comment:
Please use: `err_msg = f'Configuration error in rule {rule}: '`

Reply:
done